nebula/ui/components/MZSystemAlert.qml

```diff
@@ -72,13 +72,7 @@ MZAlert {
                     target: alertBox
                     //% "Background service error"
                     alertText: qsTrId("vpn.alert.backendServiceError")
-                    //% "Restore"
-                    //: Restore a service in case of error.
-                    alertActionText: qsTrId("vpn.alert.restore")
                     visible: true
-                    onActionPressed: ()=>{
-                        VPN.backendServiceRestore();
```

Comment:
`MozillaVPN::backendServiceRestore()` is still reachable through a few other code paths when handling other errors, but there is a reasonable case to remove it, because none of it has ever done anything useful.

Reply:
It has been removed.